SPECS/azurelinux-repos/azurelinux-repos.spec

```diff
@@ -25,12 +25,15 @@ Source15:       azurelinux-amd-preview.repo
 
 Requires:       %{name}-shared = %{version}-%{release}
 
+%ifarch x86_64
```

Comment:
Looks good. Not a blocker, just a kind request: could you add a line of comment explaining that we can't use `ifarch` directly, because using that conditional after `BuildArch: noarch` doesn't work? I'm pretty sure someone's going to look at this later and try to understand what's going on.

Reply:
yep that's probably going to me next week :)  thanks!
Does this cover it?